src/ios/Camera.strings

```diff
@@ -0,0 +1,3 @@
+"camera.prohibited" = "Access to the camera has been prohibited; please enable it in the Settings app to continue.";
+"camera.ok" = "Settings";
+"camera.settings" = "OK";
```

Comment:
These last two strings seem to be swapped

Reply:
Oh sorry about that should I update ?

Reply:
Yes.

Reply:
Done